sponsoring.html

```diff
@@ -35,6 +35,22 @@ <h3 style="margin-top: 30px;">About the Conference</h3>
                 <li>Video recordings of all talks</li>
             </ul>
 
+            <h3 style="margin-top: 30px;">Why else aside from the package contents should you sponsor?</h3>
```

Comment:
We have so many `style="margin-top: 30px;"` here, I think it would be nicer to put that into our css. What do you think? :)

Reply:
I agree, but don't trust myself to try and extract the style without breaking everything..

Reply:
Ah, ok. So then we'll merge it and beautify it later :)